test/Interop/Cxx/templates/Inputs/decl-with-definition.h

```diff
@@ -0,0 +1,25 @@
+#ifndef TEST_INTEROP_CXX_TEMPLATES_INPUTS_DECL_WITH_DEFINITION_H
+#define TEST_INTEROP_CXX_TEMPLATES_INPUTS_DECL_WITH_DEFINITION_H
+
+template<class T>
+struct MagicWrapper {
+  T t;
+  int callGetInt() const {
+    return t.getInt() + 5;
+  }
+};
+
+struct MagicNumber {
+  int getInt() const { return 24; }
+};
+
+inline MagicWrapper<MagicNumber> forceInstantiating() {
+  return MagicWrapper<MagicNumber>();
+}
+
+// MagicWrapper<MagicNumber> ClassTemplateSpecializationDecl has definition
+// because function above forced the instantiation. Its members are not
+// instantiated though, we need to instantiate them in Swift.
```

Comment:
```suggestion
// MagicWrapper<MagicNumber> ClassTemplateSpecializationDecl has a definition
// because function above forced the instantiation. Its members are not
// instantiated though, the Swift compiler needs to instantiate them.
```